doc/Makefile.am

```diff
@@ -395,6 +395,10 @@ EXTRA_DIST = \
 	index.rst \
 	index_man.rst \
 	$(RST_FILES) \
+	man1/index.rst \
+	man1/index_general.rst \
+	man1/index_job.rst \
+	man1/index_kvs.rst \
```

Comment:
Since this moves the man pages around, will the link(s) in flux-docs need to be updated? (I actually don't know)

Reply:
Oh good question.  Well the man pages didn't move in the repo - only the toctrees changed.   For example:

https://flux-framework--5470.org.readthedocs.build/projects/flux-core/en/5470/man1/flux-job.html

So I _think_ we're OK!

Reply:
Oh yeah, also as long as this link still works:

https://flux-framework.readthedocs.io/projects/flux-core/en/latest/index.html#man-pages

Then we're fine. Thanks

Reply:
Hmm, I'm not sure now that I look at it.  This goes to the landing page:

https://flux-framework--5470.org.readthedocs.build/projects/flux-core/en/5470/index.html#man-pages

But the link you mentioned (presumably from [here](https://flux-framework.readthedocs.io/en/latest/#manual-pages)) is an intersphinx reference 
```rst
:ref:`core:man-pages`
```
and we still define that label in `index_man.rst`:
```rst
.. _man-pages:

Manual Pages
============

.. toctree::
   :maxdepth: 1

   man1/index
   man3/index
   man5/index
   man7/index
```
I would kind of expect some magic to happen when that URI is resolved but it doesn't seem to be happening in the preview.

Reply:
The link isn't _broken_ per se since the anchor is ignored if it doesn't exist, and the link does land on a page where the man pages are conveniently accessible in the left column table of contents.

We can update the link in flux-docs but of course we don't know what else has cached the link in question.

What do you think @grondo - is this reasonable to merge as is?

Reply:
OK, I'll set MWP and if we broken anything we can go from there.  This is not exactly an area I know much about!